tests/test_annotations.py

```diff
@@ -487,10 +518,6 @@ class C:
             y = attr.ib(type=str)
             z = attr.ib()
 
-        assert "int" == attr.fields(C).x.type
```

Comment:
Why did you remove these?

Reply:
Because they don't work across Python versions (they become typing.ForwardRef or something like that). I figured it's more important it ultimately gives us what we need?